app/models/spreadsheets/local_ngo_spreadsheet.rb

```diff
@@ -3,8 +3,8 @@
 class Spreadsheets::LocalNgoSpreadsheet
   attr_reader :program
 
-  def initialize(program_id)
-    @program = Program.find(program_id)
+  def initialize(program_uuid)
```

Comment:
@hengsokly why don't we pass program object from the outside instead of passing program_uuid then query for the program object as it produce a complication of the dependencies injection that difficult to mock/stub in testing